events/RDS/StartExportTask.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1537 - Transfer Data to Cloud Account"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1567 - Exfiltration Over Web Service",
+            "reason": "By exporting data to an S3 bucket, adversaries can use cloud services as a method to exfiltrate data without direct interaction with the database."
+        },
+        {
+            "technique": "T1530 - Data from Cloud Storage Object",
+            "reason": "After exporting snapshot data to S3, an adversary can retrieve and analyze the data from the S3 bucket, provided they maintain access to the cloud storage."
+        },
+        {
+            "technique": "T1552 - Unsecured Credentials",
+            "reason": "If the database snapshots contain unsecured credentials, such as plaintext passwords or API keys, exporting the snapshot to S3 could allow an adversary to access these credentials, which could then be used to further compromise the environment or exfiltrate additional data."
+        },
+        {
+            "technique": "T1213 - Data from Information Repositories",
+            "reason": "By exporting the RDS snapshot, an adversary gains access to a repository of information stored within the database, which they can then access through the S3 bucket."
+        },
+        {
+            "technique": "T1078 - Cloud Accounts",
+            "reason": "Adversaries may leverage compromised cloud accounts to persist within the environment, using cloud-native functionality like the StartExportTask to maintain access to sensitive data over time."
+        },
+        {
+            "technique": "T1556 - Credentials from Password Stores",
+            "reason": "If the database snapshots contain password hashes or other credential stores, exporting the snapshot to S3 could allow an adversary to collect and crack these credentials for further exploitation."
```

Comment:
Similar to Unsecured Credentials, I don't see this related. I agree that after a copy you can probably access and extract sensitive data ( maybe credentials ) but I find it difficult to match with the StartExportTask

Reply:
Removed.